.solhint.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "solhint:recommended",
   "rules": {
-    "compiler-version": ["error", "^0.8.0"],
+    "compiler-version": ["warn", "^0.8.0"],
```

Comment:
Are we able to disable this entirely?  I know some novice users can become confused by these version warnings and I do not believe they are very important for the vast majority of users.